client/my-sites/google-my-business/stats/chart.js

```diff
@@ -0,0 +1,122 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { isEqual, get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import FakeData from './fake-data';
+import PieChart from 'components/pie-chart';
+import PieChartLegend from 'components/pie-chart/legend';
+import SectionHeader from 'components/section-header';
+import { changeGoogleMyBusinessStatsInterval } from 'state/google-my-business/actions';
+import { getInterval } from 'state/google-my-business/selector';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+class GoogleMyBusinessStatsChart extends Component {
+	static propTypes = {
+		changeGoogleMyBusinessStatsInterval: PropTypes.func.isRequired,
+		chartTitle: PropTypes.oneOfType( [ PropTypes.func, PropTypes.string ] ),
+		data: PropTypes.array.isRequired,
+		dataSeriesInfo: PropTypes.object,
+		description: PropTypes.string,
+		interval: PropTypes.oneOf( [ 'week', 'month', 'quarter' ] ),
+		siteId: PropTypes.number.isRequired,
+		statType: PropTypes.string.isRequired,
+		title: PropTypes.string.isRequired,
+	};
+
+	static defaultProps = {
+		dataSeriesInfo: {},
+	};
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			transformedData: this.transformData( props.data ),
+		};
+	}
```

Comment:
As mentioned by @ockham a few days ago [in a different pull request](https://github.com/Automattic/wp-calypso/pull/24044#discussion_r180825102), we should be able to get rid of such constructor and save a few lines with:

```javascript
state = {
  transformedData: this.transformData( this.props.data ),
};
```